cmd/ciReleaseDelete.go

```diff
@@ -90,6 +92,18 @@ var ciReleaseDeleteCmd = &cobra.Command{
 			for _, v := range list.Items {
 				PVC_client.Delete(context.TODO(), v.Name, v1.DeleteOptions{})
 			}
+
+			// Remove based on app.kubernetes.io/instance label
+			list, err = PVC_client.List(context.TODO(), v1.ListOptions{
+				LabelSelector: "release=" + releaseName,
```

Comment:
Forgot to update the label here to "app.kubernetes.io/instance=" :)

Reply:
Furthermore, could we reduce the duplicate lines of code by wrapping this logic into a loop, which loops through the LabelSelectors?

Reply:
Good catch, thank You! Fixed that and did a slight de-duplication.